inputfiles/overridingTypes.json

```diff
@@ -129,6 +129,18 @@
                     "property": {
                         "onerror": {
                             "override-type": "OnErrorEventHandler"
+                        },
+                        "ontouchcancel": {
+                            "nullable": 1
```

Comment:
I got an ugly working hack: Try replacing this line with `"name": "ontouchcancel?"` and so on.

Reply:
Hmm, I could have sworn that it was not a no-op when I committed...

Reply:
Ewe.... that is dirty... but it appears to work.  Pushed.